packages/kbn-esql-ast/src/ast_parser.ts

```diff
@@ -25,5 +29,9 @@ export function getAstAndSyntaxErrors(text: string | undefined): {
 
   parser[ROOT_STATEMENT]();
 
-  return { ...parseListener.getAst(), errors: errorListener.getErrors() };
+  const errors = errorListener.getErrors().filter((error) => {
+    return !SYNTAX_ERRORS_TO_IGNORE.includes(error.message);
+  });
+
+  return { ...parseListener.getAst(), errors };
```

Comment:
RFC: is this the right level to ignore the error at? If we only want these ignored in the editor, we could always do this at a higher level in the architecture.